.github/workflows/ci-unix-static.yml

```diff
@@ -51,6 +51,21 @@ jobs:
       with:
         toolchain: stable
 
-    # Build and test CrabbyAvif
+    - name: Build and run the Rust tests
+      run: cargo test -- --skip test_conformance
 
-    - run: cargo test -- --skip test_conformance
+    - name: Build GoogleTest
+      if: steps.cache-ext.outputs.cache-hit != 'true'
+      working-directory: ./third_party
+      run: bash -e googletest.cmd
+
+    - name: Build and run the C++ tests
+      # TODO: This step fails on macos. So run it on linux only for now.
+      if: runner.os == 'Linux'
+      run: |
+        cargo build --features capi --release
+        mkdir c_build
+        cd c_build
+        cmake ../c_api_tests/
+        make
```

Comment:
To avoid `cd` we could do

```sh
cmake -S c_api_tests -B c_build
cmake --build c_build --parallel
```

but there is the `make test` line anyway.

Reply:
Ack. yeah the make test line is there so we would have to cd anyways.

Reply:
make has -C for this.

Reply:
Done in https://github.com/webmproject/CrabbyAvif/pull/30